### DISCUSSION TITLE
Your own print function in the logger

### DISCUSSION THREAD

**Discussion by 1AlexFix1** [dsrenesanse/worker_manager#90] [Ideas]
Hello! 
We would be very grateful if it were possible to override the log function.

**Comment by dsrenesanse** [comment]
this one?
![Screenshot 2024-04-19 at 11 49 23 AM](https://github.com/machinescream/worker_manager/assets/42138594/6ec1611f-c576-45b1-9f9f-90e13f5b9369)
